digicampipe/tests/test_digicamtoy_event_stream.py

```diff
@@ -0,0 +1,59 @@
+from digicampipe.io.hdf5 import digicamtoy_event_source
+from digicampipe.io.event_stream import event_stream
+import pkg_resources
+import os
+import pytest
+
+example_file_paths = []
+
+for i in range(12):
+
+    path = pkg_resources.resource_filename(
+        'digicampipe',
+        os.path.join(
+            'tests',
+            'resources',
+            'digicamtoy',
+            'test_digicamtoy_{}.hdf5'.format(i)
+        )
+    )
+
+    example_file_paths.append(path)
+
+TEL_WITH_DATA = 1
+N_PIXELS = 1296
+
+
+def test_event_source():
+
+    for _ in digicamtoy_event_source(example_file_paths[0]):
+        pass
+
+
+def test_event_stream():
+
+    for _ in event_stream(example_file_paths, mc=True):
```

Comment:
the `mc` option was dropped from event_stream. event stream checks the filename to give you the right event_source.

Reply:
This is used to guess the event_source

https://github.com/cta-sst-1m/digicampipe/blob/7db37445b6fe32efdc5e5448f8c0cd877b363ee0/digicampipe/io/event_stream.py#L37-L43

Reply:
thanks